src/mutation_signatures_visualization.py

```diff
@@ -47,6 +47,33 @@
 ################################################################################
 # PLOTS
 ################################################################################
+
+# counts_df is samples-by-categories
+# exposures_df samples-by-signatures
+# signatures_df is signatures-by-categories
+def plot_signatures(counts_df, signature_df, exposure_df, output_file):
```

Comment:
Add `palette` and `ylabel` as optional arguments

Reply:
More specifically, try to make the function call match the `sbs_signature_plot` (since you are eventually calling tht)